src/test/unit/lib/model/OutboundTransfersModel.test.js

```diff
@@ -215,6 +216,98 @@ describe('outboundModel', () => {
     });
 
 
+    test('uses quote response transfer amount for transfer prepare', async () => {
+        defaultEnv.AUTO_ACCEPT_PARTY = 'true';
+        defaultEnv.AUTO_ACCEPT_QUOTES = 'true';
+
+        await setConfig(defaultEnv);
+        const conf = getConfig();
+        const cache = new Cache();
+
+        MojaloopRequests.__getParties = jest.fn(() => {
+            cache.emitMessage(JSON.stringify(payeeParty));
+            return Promise.resolve();
+        });
+
+        const dummyTransferAmount = {
+            currency: 'XYZ',
+            amount: '9876543210'
+        };
+
+        MojaloopRequests.__postQuotes = jest.fn(() => {
+            // ensure that the `MojaloopRequests.postQuotes` method has been called with correct arguments
+            // including extension list
+            expect(MojaloopRequests.__postQuotes).toHaveBeenCalledTimes(1);
```

Comment:
I think this should placed outside of the function body, after `await model.run();`